SUPPORT.md

```diff
@@ -5,7 +5,7 @@ Types of questions and where to ask:
 - How do I? -- the Discourse forum at [community.k6.io](https://community.k6.io/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/k6) (use tags: k6, javascript, load-testing)
 - I got this error, why? -- [community.k6.io](https://community.k6.io/) or [Stack Overflow](https://stackoverflow.com/questions/tagged/k6)
 - I got this error and I'm sure it's a bug -- [open a new issue](https://github.com/k6io/k6/issues), if there isn't one for this specific bug already
```

Comment:
```suggestion
- I got this error and I'm sure it's a bug -- [open a new issue](https://github.com/grafana/k6/issues), if there isn't one for this specific bug already
```

Reply:
Oops...missed that one! Nice catch, thank you!